### ISSUE TITLE
Hint Header Error in TeX Mode

### ISSUE THREAD

**Issue by maharper** [openwebwork/pg#664]
There is a TeX mode error when preparing hardcopy containing an "Instructor hint preview".

Line 1278 of PGbasicmacros.pl reads
`" ", $showHint + 1, ")", $EITALIC, "\\par\\smallskip", @in);`
when @in starts with an alphanumeric it runs together with smallskip creating a TeX error.
For example, if the Hint starts "Try the ...", `\smallskipTry` will get written into the TeX code and an error produced:
```
! Undefined control sequence.
l.265 ...number of attempts: 2)} \par\smallskipTry
                                                   the 
The control sequence at the end of the top line
of your error message was never \def'ed.
```
Adding a space after smallskip seems to fix it:
l 1278: `" ", $showHint + 1, ")", $EITALIC, "\\par\\smallskip ", @in);`

**Comment by drgrice1** [comment]
Do you have a minimal working example for which this occurs?  I am unable to reproduce this.  Everything compiles fine for me even if there is a hint that begins with alpha-numeric characters.

**Comment by maharper** [comment]
I can cook one up.  The instructor hint preview has to be triggered.  It is not enough to just have a hint in the question.

**Comment by drgrice1** [comment]
I see.  So it happens with problems that do not use `PGML`, and instead use the old basic pg `BEGIN_HINT` or really old `HINT(EV3(<<'END_HINT'));` aproaches.  If you are writing new problems, then you should switch to using `PGML`.  Newly written problems should not use the basic pg methods.

We will add the space for the next release.

**Comment by maharper** [comment]
Yes.  All my authoring is now using PGML.  I noticed this with an old problem (from the OPL) that uses BEGIN_HINT/END_HINT.  Thanks.

**Comment by maharper** [comment]
This is fixed by #665 I think it can be closed. Thanks.
